reference/api/tasks.md

```diff
@@ -61,17 +67,17 @@ This value is always `null` for `dumpCreation` tasks.
 
 #### `documentAdditionOrUpdate`
 
-| Name                    | Description                  |
-| :---------------------- | :--------------------------- |
-| **`receivedDocuments`** | Number of documents received |
-| **`indexedDocuments`**  | Number of documents indexed  |
+| Name                    | Description                                                                            |
+| :---------------------- | :------------------------------------------------------------------------------------- |
+| **`receivedDocuments`** | Number of documents received                                                           |
+| **`indexedDocuments`**  | Number of documents indexed. `null` when the task status is `enqueued` or `processing` |
 
 #### `documentDeletion`
 
-| Name                      | Description                     |
-| :------------------------ | :------------------------------ |
-| **`receivedDocumentIds`** | Number of document ids received |
-| **`deletedDocuments`**    | Number of documents deleted     |
+| Name                   | Description                                                                            |
+| :--------------------- | :------------------------------------------------------------------------------------- |
+| **`matchedDocuments`** | Number of document to be deleted based on the request                                 |
+| **`deletedDocuments`** | Number of documents deleted. `null` when the task status is `enqueued` or `processing` |
```

Comment:
There's something I find confusing about the way we're describing these fields—it's very easy to miss the difference between "to be deleted" and "deleted". Not sure it can be helped, but here's an attempt.

```suggestion
| **`matchedDocuments`** | Number of documents queued for deletion                                                      |
| **`deletedDocuments`** | Number of documents deleted. `null` while the task status is `enqueued` or `processing` |
```